src/flamenco/runtime/fd_runtime.h

```diff
@@ -385,6 +385,28 @@ fd_runtime_microblock_verify_ticks( fd_exec_slot_ctx_t *        slot_ctx,
                                     ulong              max_tick_height,
                                     ulong              hashes_per_tick );
 
+/*
+   Verify that a list of txns (e.g., those in a microblock) do not have read-write or write-write conflits
+ */
+struct fd_txn_writes {
+  fd_acct_addr_t key;
+  ulong next;
+};
+typedef struct fd_txn_writes fd_txn_writes_t;
+#define FD_TXN_CONFLICT_MAP_SEED      (0UL)
+
+#define FD_TXN_CONFLICT_OK            (0UL)
+#define FD_TXN_CONFLICT_READ_WRITE    (1UL)
+#define FD_TXN_CONFLICT_WRITE_WRITE   (2UL)
+#define FD_TXN_CONFLICT_MAP_TOO_SMALL (3UL)
```

Comment:
can you change this to follow this convention (use `int`) and negative err values

`FD_RUNTIME_SUCCESS                     ( 0)`
`FD_RUNTIME_ERR_TXN_CONFLICT_RW (-1)`